tooling/lsp/src/requests/references.rs

```diff
@@ -94,6 +94,9 @@ mod references_tests {
         check_references_succeeds("rename_function", "another_function", 0, false).await;
     }
 
+    // Ignored because making this work slows down everything, so for now things will not work
+    // as ideally, but they'll be fast.
```

Comment:
Do we want to bring this feature back? If so, let's make an issue and link it here.